terminal_reader.go

```diff
@@ -0,0 +1,47 @@
+package stripe
+
+// TerminalReaderRegisterParams is the set of parameters that can be used for creating a terminal reader.
+type TerminalReaderRegisterParams struct {
```

Comment:
Convention throughout the library is to keep the parameters for "new" and "update" in the same struct — let's combine this with `TerminalReaderUpdateParams`.